packages/indy-vdr/src/index.ts

```diff
@@ -0,0 +1,5 @@
+try {
+  require('indy-vdr-test-nodejs')
```

Comment:
How will this work when we want to use this package in React Native? Should we add the require('indy-vdr-test-react-native') to the catch block?

Reply:
I was thinking we could either do it with the catch as you say, or we detect the platform. Or would you prefer the manual approach? (so you provide it to the module). That's also fine for me.